js/libs/ui-shared/src/main.ts

```diff
@@ -24,6 +24,7 @@ export {
 export { IconMapper } from "./icons/IconMapper";
 export { FormPanel } from "./scroll-form/FormPanel";
 export { ScrollForm, mainPageContentId } from "./scroll-form/ScrollForm";
+export { FormSubmitButton } from "./buttons/FormSubmitButton";
```

Comment:
Let's also export the props here, so they can be re-used when necessary. 
```suggestion
export { FormSubmitButton, type FormSubmitButtonProps } from "./buttons/FormSubmitButton";
```